modules/ossm-security-mtls.adoc

```diff
@@ -99,3 +120,19 @@ The default is `TLS_AUTO` and does not specify a version of TLS.
 |`TLSv1_3`
 |TLS version 1.3
 |===
+
+[id="ossm-security-enabling-controlplane_{context}"]
```

Comment:
Should we be documenting Mixer settings in 2.0?  Should this be 1.1 content?

Reply:
Yes, this is specifically for the deprecated format of Mixer. However, it further explains the difference between configuring mtls in the control plane and the data plane, which is valuable on its own, even if Mixer is deprecated.